### PR TITLE
Feature/branded gui

### PR DESCRIPTION
**Pull request by dishmop**
## Proposed changes

GUi is now branded according to Fetch.ai guidelines. 
Registry panel in gui is now search based (rather than listing everything)
Minor usability improvements (e.g. curse shows when things are intractable and buttons flash when clicked)

## Fixes

-
## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../master/CONTRIBUTING.rst) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x ] I have checked that code coverage does not decrease.
- [x] I have checked that the documentation about the `aea cli` tool works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/test_gui/test_misc.py [hunk 1]
```diff
@@ -52,6 +52,18 @@ def test_icon():
     assert result.status_code == 200
 
 
+def test_logo():
+    """Test that the home-page exits."""
+    app = create_app()
+
+    # sends HTTP GET request to the application
+    # on the specified path
+    result = app.get('/logo.png')
```
Comment (DavidMinarsch) [inline tests/test_gui/test_misc.py]: @dishmop has this endpoint not been removed with the last commit?

File: aea/cli_gui/__init__.py [hunk 2]
```diff
@@ -435,6 +435,12 @@ def favicon():
         return flask.send_from_directory(
             os.path.join(app.root_path, 'static'), 'favicon.ico', mimetype='image/vnd.microsoft.icon')
 
+    @app.route('/logo.png')
+    def logo():
```
[the remark this answers is not in the document]
Comment (dishmop) [reply aea/cli_gui/__init__.py]: fixed

File: aea/cli_gui/templates/home.js [hunk 3]
```diff
@@ -425,17 +425,13 @@ class Controller{
...
-                    var text = test.text;
-                    var val = test.value;
-                    $(this).css("background-color", "gray");
-                })
-
-
-
-
-
+//                $(this).children().each(function(i) {
```
[the remark this answers is not in the document]
Comment (dishmop) [reply aea/cli_gui/templates/home.js]: done

File: tests/test_gui/test_misc.py (hunk 1 above)
Comment (dishmop) [reply tests/test_gui/test_misc.py]: Yes - there is a different end point where the logo now sits, but the code doesn't know anything about it (it is just loaded directly by the html).

**Pull request merged**